.ci/Jenkinsfile

```diff
@@ -322,8 +322,15 @@ def generateFunctionalTestStep(Map args = [:]){
   }
   tags += excludeNightlyTag
 
+  workerLabels = "${platform}"
+  if (platform != "arm") {
+    // ARM is not ready as immutable workers
+    workerLabels += " && immutable"
+  }
+  workerLabels += " && docker"
```

Comment:
* https://beats-ci.elastic.co/label/arm&&docker/
* https://beats-ci.elastic.co/label/immutable&&docker/

Reply:
Will add this!

Reply:
Mmm... does it mean that we cannot use `immutable` workers on ARM? If that's the case, the logic here is correct, right? we are only adding the immutable label for `!arm` workers.